src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,14 +2,18 @@
 
 ## Unreleased
 
+* Added `net.host.name` and `net.host.port` dimensions to `http.server.duration`
+metric.
+[#3928](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3928)
+
+* **Breaking change** The `http.server.duration` metric's `http.target`
+attribute is replaced with `http.route` attribute.
+([#3903](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3903))
+
```

Comment:
These entries could be reorganized to make it more readable. Something like this:
* Users migrating from `1.0.0-rc9.9` to this version will see the following breaking changes:
   -
   -
   -

Reply:
Updated